### ISSUE TITLE
Preprocessing issues of the MeetRoom dataset

### ISSUE THREAD

**Issue by Ataraxia12** [WuJH2001/swift4d#15]
Thank you for sharing your MeetRoom dataset preprocessing code. I found that when using convert.by to process scene trimming, the /data/Meetroom/trimming/frame00000/images folder does not contain all images, even after multiple attempts. Do you have any suggestions on this? By the way, scene discussion can be processed correctly.

**Comment by Ataraxia12** [comment]
If it's convenient, could you please share the preprocessed MeetRoom dataset directly? Thank you very much!

**Comment by WuJH2001** [comment]
> If it's convenient, could you please share the preprocessed MeetRoom dataset directly? Thank you very much!

Hi @Ataraxia12  COLMAP has difficulty extracting all viewpoint data for the MeetRoom dataset. We also had to try more than a dozen times to obtain a complete reconstruction. We will update the initial point cloud for MeetRoom later tonight.

**Comment by Ataraxia12** [comment]
After processing the discussion dataset, I attempted to reconstruct it. I used the Cam0 view as the test set and the remaining views for training. I found that during the training process, the PSNR of the test set remained around 19, while the PSNR of the training set increased to around 30. Why is there such a serious overfitting? Is it related to the data preprocessing results?

<img width="1280" height="720" alt="Image" src="https://github.com/user-attachments/assets/78212247-8637-46d3-82ab-7c7552882b41" />

**Comment by WuJH2001** [comment]
Hi @Ataraxia12 , The MeetRoom dataset is characterized by sparse geometry and weak textures, which makes it challenging for COLMAP to perform accurate camera calibration. This often leads to overfitting or suboptimal results. In practice, it usually requires running COLMAP multiple times to obtain a reliable set of camera parameters. We also had to run it several times before achieving a reasonable result. So far, I have only found the camera parameters for the "Discussion" scene [here](https://1drv.ms/f/c/80737028a7921b70/EqXgyFgTWQBFgEZrTZK_3TwB9607YkS6VYrNP2ijADCK8Q?e=PXne64); I'm still searching for the others.

**Comment by Ataraxia12** [comment]
I found that when combining the preprocessed camera poses you provided and using convert_frames.py to process the images of other frames, the size of the undistorted images for other frames remains 720×1270, while the image size in frame000000/images is 684×1222.

**Comment by WuJH2001** [comment]
> I think when processing the meetroom dataset, the --min_scale 1 --max_scale 1 settings in the colmap command need to be removed.

You need to copy the camera parameters provided for frame 0 to the other frames before performing undistortion (copy_cams.py). The undistortion process will change the image resolution, as you pointed out—resulting in 684×1222.
